common/protob/messages-cardano.proto

```diff
@@ -218,6 +218,7 @@ message CardanoSignTxInit {
     required bool has_collateral_return = 19;
     optional uint64 total_collateral = 20;
     required uint32 reference_inputs_count = 21;
+    required bool has_output_details = 22;                      // used to hide initial screen for ordinary transactions if possible
```

Comment:
Done and done. https://github.com/vacuumlabs/trezor-firmware/pull/79/commits/eddc71d2c1978a113e8acefe7ac1f7c77d413067 I was sceptical of using a general name so that it would be obvious what it means but yeah, it's probably more future proof to use a general name.